azure-pipelines.perf-test.yml

```diff
@@ -154,3 +158,52 @@ jobs:
           uniqueId: 'perfComment9423'
 
       - template: .devops/templates/cleanup.yml
+
+  - job: PerfTestReactComponents
+    displayName: Perf test for @fluentui/react-components
+    dependsOn: CheckIfPackagesAffected
+    condition: eq(dependencies.CheckIfPackagesAffected.outputs['PackagesAffected.ReactComponentsPackageAffected'], true)
+    workspace:
+      clean: all
+    steps:
+      - template: .devops/templates/tools.yml
+      - task: Bash@3
+        inputs:
+          filePath: yarn-ci.sh
+        displayName: yarn
+
+      # Build v8 Perf Test
```

Comment:
```suggestion
      # Build v9 Perf Test
```

Reply:
maybe we can remove these comments completely now when the jobs are decoupled ?

Reply:
Will remove the comments